sqlglot/tokens.py

```diff
@@ -11,6 +12,14 @@
     from sqlglot.dialects.dialect import DialectType
 
 
+try:
+    import sqlglotrs  # noqa
```

Comment:
Nit: should we do `# type: ignore` instead here, similar to https://github.com/tobymao/sqlglot/blob/main/sqlglot/tokens.py#L1260? I don't see `noqa` being used anywhere else in the codebase (does it serve a different purpose?), so this comment is mostly for improving consistency.

Reply:
Please note that I didn’t do this because of mypy, but because autoflake would delete an unused import. 

# type applies to mypy only afaik, but will try your suggestion anyway

Reply:
Ahh that makes sense, thanks for clarifying. Feel free to ignore then.

Reply:
Actually `type: ignore` did seem to work. So I was wrong about my assumption. Thanks!

Reply:
Cool!